### PR TITLE
Update parser docs for plugins enabled by default since 7.14.0

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Update parser docs for https://github.com/babel/babel/pull/13175

### CODE REVIEW

File: docs/parser.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ require("@babel/parser").parse("code", {
   <summary>History</summary>
 | Version | Changes |
 | --- | --- |
-| `v7.14.0` | Added `asyncDoExpressions` |
+| `v7.14.0` | Added `asyncDoExpressions` Move `classProperties`, `classPrivateProperties`, `classPrivateMethods` to Latest ECMAScript features |
```
Comment (nicolo-ribaudo) [inline docs/parser.md:187]: Also `moduleStringNames`

**Comment by JLHwung** [comment]
Oh missed this one, can you rebase? https://github.com/babel/website/pull/2518 is merged.

**Comment by sosukesuzuki** [comment]
fixed!

File: docs/parser.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ require("@babel/parser").parse("code", {
   <summary>History</summary>
 | Version | Changes |
 | --- | --- |
-| `v7.14.0` | Added `asyncDoExpressions` |
+| `v7.14.0` | Added `asyncDoExpressions` Move `classProperties`, `classPrivateProperties`, `classPrivateMethods`, `moduleStringNames` to Latest ECMAScript features |
```
Comment (JLHwung) [inline docs/parser.md:187]:
```suggestion
| `v7.14.0` | Added `asyncDoExpressions`. Moved `classProperties`, `classPrivateProperties`, `classPrivateMethods`, `moduleStringNames` to Latest ECMAScript features |
```

**Pull request merged**